applications/rag/main.tf

```diff
@@ -77,28 +77,28 @@ module "kuberay-operator" {
 }
 
 module "gcs" {
-  source      = "../../modules/gcs"
+  source      = "github.com/GoogleCloudPlatform/ai-on-gke//modules/gcs"
+  count       = var.create_gcs_bucket ? 1 : 0
```

Comment:
Where is this var added? Missing from https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/modules/gcs/variables.tf

It should be added to the GCS module and defaulted to true in all 3 applications